dev/tasks/python-wheels/github.win.yml

```diff
@@ -53,9 +53,9 @@ jobs:
         run: |
           echo "c:\\Miniconda\\condabin" >> $GITHUB_PATH
       - name: Build wheel
-        shell: bash
+        shell: cmd
         run: |
-          arrow/dev/tasks/python-wheels/win-build.bat
+          call arrow/dev/tasks/python-wheels/win-build.bat
```

Comment:
Sorry, could you revert this change by me? https://github.com/apache/arrow/pull/8386/commits/5e03e010bc65308452c95d51ed21f0baebebf059
This was meaningless.

Reply:
Ok, I left it in because I wasn't sure if it actually allowed the error code to be used, I'll revert since it sounds like it wasn't a problem